ansible/library/exabgp.py

```diff
@@ -236,21 +240,22 @@ def main():
     local_asn = module.params['local_asn']
     peer_asn  = module.params['peer_asn']
     port      = module.params['port']
+    dump_script = module.params['dump_script']
 
-    setup_exabgp_processor()
+    setup_exabgp_processor(dump_script==DEFAULT_DUMP_SCRIPT)
```

Comment:
The argument name is not dump_script,  but default_dump_script
As I see it here: 
https://github.com/Azure/sonic-mgmt/pull/2553/files#diff-ca868a64bc6e0673c7630841edad82bd6de93e60ea7a7ba694c87467d69c8f2eR209